lib/features/manage_account/presentation/email_rules/widgets/email_rule_item_widget.dart

```diff
@@ -23,47 +40,68 @@ class EmailRulesItemWidget extends StatelessWidget {
       padding: EdgeInsets.only(
         top: 15,
         bottom: 15,
-        left: _responsiveUtils.isMobile(context) ? 16 : 24,
-        right: _responsiveUtils.isMobile(context) ? 0 : 24
+        left: responsiveUtils.isMobile(context) ? 16 : 24,
+        right: responsiveUtils.isMobile(context) ? 0 : 24
       ),
       color: Colors.white,
       child: Row(crossAxisAlignment: CrossAxisAlignment.center, children: [
-        Text(rule.name,
-            style: const TextStyle(
-                fontSize: 16,
-                fontWeight: FontWeight.w400,
-                color: Colors.black)),
+        Text(
+          rule.name,
+          style: Theme.of(context).textTheme.labelSmall?.copyWith(
+            fontSize: 16,
+            fontWeight: FontWeight.w400,
+            color: Colors.black
+          )
+        ),
+        if (_isMailboxAppendInNotExist)
+          TMailButtonWidget.fromIcon(
+            icon: imagePaths.icQuotasWarning,
+            iconSize: 20,
+            margin: const EdgeInsetsDirectional.only(start: 4),
+            padding: const EdgeInsets.all(3),
+            iconColor: AppColor.colorBackgroundQuotasWarning,
+            tooltipMessage: AppLocalizations.of(context).warningRuleCannotAppliedWhenTargetFolderNoExist,
+            backgroundColor: Colors.transparent,
+          ),
         const Spacer(),
-        if (!_responsiveUtils.isMobile(context))
+        if (!responsiveUtils.isMobile(context))
           buildIconWeb(
               icon: SvgPicture.asset(
-                _imagePaths.icEditRule,
+                imagePaths.icEditRule,
                 fit: BoxFit.fill,
                 colorFilter: AppColor.primaryColor.asFilter(),
               ),
               onTap: () {
-                _emailRuleController.editEmailRule(context, rule);
+                editRuleAction?.call(context, rule);
               }),
-        if (!_responsiveUtils.isMobile(context))
+        if (!responsiveUtils.isMobile(context))
           buildIconWeb(
               icon: SvgPicture.asset(
-                _imagePaths.icDeleteRule,
+                imagePaths.icDeleteRule,
                 fit: BoxFit.fill,
               ),
               onTap: () {
-                _emailRuleController.deleteEmailRule(context, rule);
+                deleteRuleAction?.call(context, rule);
               }),
-        if (_responsiveUtils.isMobile(context))
+        if (responsiveUtils.isMobile(context))
           buildIconWeb(
               icon: SvgPicture.asset(
-                _imagePaths.icOpenEditRule,
+                imagePaths.icOpenEditRule,
                 fit: BoxFit.fill,
               ),
               iconPadding: const EdgeInsets.all(0),
               onTap: () {
-                _emailRuleController.openEditRuleMenuAction(context, rule);
+                openEditRuleMenuAction?.call(context, rule);
               }),
```

Comment:
Adjust indent

Reply:
I don't think we need to change it from the old code.